docs/languages/Java.md

```diff
@@ -31,7 +31,9 @@ TODO
 TODO
 
 ## Include javax validation constraints annotations for properties
-TODO
+In some cases where you generate the models from JSON Schema and you might want them to include `javax.validation.constraint` annotations.
+
+Check out this [example out for a live demonstration](../../examples/java-generate-javax-constraint-annotation).
 
 ## Include Jackson annotations for the class
```

Comment:
Jackson?

Reply:
See this @alequetzalli 🙂 -> https://github.com/asyncapi/modelina/issues/427